src/main/java/com/microsoft/graph/requests/GraphServiceClient.java

```diff
@@ -129,13 +131,28 @@ public String getServiceSDKVersion() {
      * @return builder to start configuring the client
      */
     @Nonnull
-    public static Builder<OkHttpClient> builder() {
+    public static Builder<OkHttpClient, Request> builder() {
+        return builder(OkHttpClient.class, Request.class);
+    }
+
+    /**
+     * Gets the builder to start configuring the client
+     *
+     * @param <nativeClient> the type of the native http client
+     * @param <nativeRequest> the type of the native http request
+     * @param nativeClientClass the class of the native http client
+     * @param nativeRequestClass the class of the native http request
+     * @return builder to start configuring the client
+     */
+    @Nonnull
+    public static <nativeClient, nativeRequest> Builder<nativeClient, nativeRequest> builder(Class<nativeClient> nativeClientClass, Class<nativeRequest> nativeRequestClass) {
         return new Builder<>();
     }
     /**
      * Builder to help configure the Graph service client
+     * @param <NativeRequestType> type of a request for the native http client
      */
-    public static class Builder<httpClientType> extends BaseClient.Builder<httpClientType> {
+    public static class Builder<httpClientType, NativeRequestType> extends BaseClient.Builder<httpClientType, NativeRequestType> {
```

Comment:
I don't know what the convention is but the naming is inconsistent for the generics. camelCase httpClientType vs PascalCase NativeRequestType. Please convert to one.

Reply:
yep, sorry, I meant to fix that and forgot somehow